elastic.go

```diff
@@ -342,11 +385,55 @@ func restoreDocuments(sb s3Backend, c elastic.Client, keyPath, indexName string)
 				},
 			)
 			if err != nil {
-				log.Fatalf("Unexpected error: %s", err)
+				log.Errorf("Unexpected error: %s", err)
+				return err
 			}
 			i++
 		}
 	}
 
-	return err
+	return nil
+}
+
+func (es *esClient) indexDocuments(indexName string) error {
```

Comment:
This separation in different files was suggested by @viklund. This method is only used in the integration test. It is a question of taste I guess.

Reply:
I know, I would rather remove it completely as it serves no point other in testing.

Reply:
But it can be done as part of test restructuring.